Source/Core/CommandLineOptions.cs

```diff
@@ -543,7 +543,8 @@ void ObjectInvariant2()
     public int VerifySnapshots = -1;
     public bool VerifySeparately = false;
     public string PrintFile = null;
-
+    public bool EmitDebugInformation { get; set;  } = true;
```

Comment:
If this is configurable, we should also have a command-line switch for it.

Reply:
Added it.